docs/pages/reference/mesheryctl/subcommands/mesheryctl-pattern-delete.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: mesheryctl pattern delete | Meshery
+permalink: reference/mesheryctl/pattern/delete
+type: reference
+display-title: "false"
```

Comment:
Good use of this option.

Reply:
@leecalcote I think this exists in all the files present in the master branch, https://github.com/layer5io/meshery/blob/master/docs/pages/reference/mesheryctl/subcommands/mesheryctl-pattern-apply.md
This would also be taken up during the complete overhauling of meshery docs 👀